repo/fsrepo/fsrepo.go

```diff
@@ -562,7 +562,9 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 
 	var du uint64
 	err = filepath.Walk(pth, func(p string, f os.FileInfo, err error) error {
-		du += uint64(f.Size())
+		if f != nil {
```

Comment:
If there is `err`, it will get returned by the function `GetStorageUsage` anyway.